### DISCUSSION TITLE
Looking for volunteers to QA 1.2 release

### DISCUSSION THREAD

**Discussion by PJ-Finlay** [argosopentech/argos-translate#63] [General]
Version 1.2 is almost ready for release and has a number of changes. To QA install from source, try to find ways the software breaks, and create tickets if you find anything.

- [Release Notes](https://github.com/argosopentech/argos-translate/blob/9993b71e18db660c76b17b4118c16e52a61f4f35/ReleaseNotes.md)

**Comment by argosopentech** [comment]
1.2 is live on PyPI

**Reply by pierotofy** [reply]
w00t! Awesome! :clinking_glasses:

**Comment by PJ-Finlay** [comment]
1.2 is live on the Snap Store
